core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -209,14 +221,17 @@ registry.ci.openshift.org/origin/4.11:openstack-cinder-csi-driver quay.io/opensh
 registry.ci.openshift.org/origin/4.11:openstack-cinder-csi-driver-operator quay.io/openshift/origin-openstack-cinder-csi-driver-operator:4.11 quay.io/openshift/origin-openstack-cinder-csi-driver-operator:4.11.0 quay.io/openshift/origin-openstack-cinder-csi-driver-operator:latest
 registry.ci.openshift.org/origin/4.11:openstack-cinder-csi-driver-operator-test quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:4.11 quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:4.11.0 quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.11:openstack-cloud-controller-manager quay.io/openshift/origin-openstack-cloud-controller-manager:4.11 quay.io/openshift/origin-openstack-cloud-controller-manager:4.11.0 quay.io/openshift/origin-openstack-cloud-controller-manager:latest
+registry.ci.openshift.org/origin/4.11:openstack-installer quay.io/openshift/origin-openstack-installer:4.11 quay.io/openshift/origin-openstack-installer:4.11.0 quay.io/openshift/origin-openstack-installer:latest
 registry.ci.openshift.org/origin/4.11:openstack-machine-api-provider quay.io/openshift/origin-openstack-machine-api-provider:4.11 quay.io/openshift/origin-openstack-machine-api-provider:4.11.0 quay.io/openshift/origin-openstack-machine-api-provider:latest
 registry.ci.openshift.org/origin/4.11:openstack-machine-controllers quay.io/openshift/origin-openstack-machine-controllers:4.11 quay.io/openshift/origin-openstack-machine-controllers:4.11.0 quay.io/openshift/origin-openstack-machine-controllers:latest
 registry.ci.openshift.org/origin/4.11:openstack-tests quay.io/openshift/origin-openstack-tests:4.11 quay.io/openshift/origin-openstack-tests:4.11.0 quay.io/openshift/origin-openstack-tests:latest
 registry.ci.openshift.org/origin/4.11:operator-lifecycle-manager quay.io/openshift/origin-operator-lifecycle-manager:4.11 quay.io/openshift/origin-operator-lifecycle-manager:4.11.0 quay.io/openshift/origin-operator-lifecycle-manager:latest
+registry.ci.openshift.org/origin/4.11:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.11 quay.io/openshift/origin-operator-marketplace:4.11.0 quay.io/openshift/origin-operator-marketplace:latest
```

Comment:
wait, so we *don't* want images like this to be excluded?  I thought the idea was to exclude them here because they come out of OKD instead

Reply:
It works like this:
- Most images: merge -> `ocp:4.11` -> release-controller CI payload accepted -> `origin:4.11` -> quay.io/openshift/origin-...
- OKD-specific images (this PR): merge -> `origin:4.11` -> quay.io/openshift/origin-...

quay.io/openshift/origin-... images are OKD, just some of them go through OCP CI payloads and some of them need to be built in a special way. The [exclusion in release-controller configs](https://github.com/openshift/release/blob/4ce225ecb01341c3ffeb31f8489d2181d734523c/core-services/release-controller/_releases/release-ocp-4.11-ci.json#L15-L37) tell release-controller that these images should not be moved to `origin:4.11`, because they get there directly.